public/css/style.css

```
@@ -1,210 +1,218 @@
-
-    padding-top: 30px;
-    width: 600px;
-    position: fixed;
-    bottom: 10px;
-    left: 10px;
-
-    max-height: calc(100vh - 20px);
-    overflow: auto;
-
-    color: var(--default-elao-consent-dark);
-    background-color: var(--default-elao-consent-light);
-    border: solid 1px var(--default-elao-consent-neutral);
-    border-radius: 15px;
-    z-index: 2147483647;
+  --default-elao-consent-primary: var(--elao-consent-primary, #007bff);
+  --default-elao-consent-primary-dark: var(--elao-consent-primary-dark, #256cdf);
+  --default-elao-consent-secondary: var(--elao-consent-secondary, #dfebfa);
+  --default-elao-consent-secondary-dark: var(--elao-consent-secondary-dark, #cadef8);
+  --default-elao-consent-danger: var(--elao-consent-danger, #ef4055);
+  --default-elao-consent-neutral: var(--elao-consent-neutral, #eae8e8);
+  --default-elao-consent-light: var(--elao-consent-light, #fff);
+  --default-elao-consent-dark: var(--elao-consent-dark, #252525);
+
+  padding-top: 20px;
+  width: 420px;
+  height: auto;
+  position: fixed;
+  bottom: 10px;
+  left: 10px;
+  font-size: 16px;
+  font-family: Arial, Helvetica, sans-serif;
+  box-shadow: 0 0 10px rgba(0, 0, 0, .1);
+
+  max-height: calc(100vh - 20px);
+  overflow: auto;
+
+  color: var(--default-elao-consent-dark);
+  background-color: var(--default-elao-consent-light);
+  border: none;
+  border-radius: 4px;
+  z-index: 2147483647;
 }
 
 .elao-consent a {
-    color: var(--default-elao-consent-primary);
-    font-weight: bold;
-    text-decoration: underline;
+  color: var(--default-elao-consent-primary);
+  font-weight: bold;
+  text-decoration: underline;
 }
 
 .elao-consent button {
-    padding: 7px 25px;
-    color: var(--default-elao-consent-light);
-    font-weight: bold;
-    background: var(--default-elao-consent-primary);
-    border: none;
-    border-radius: 5px;
+  padding: 7px 25px;
+  color: var(--default-elao-consent-light);
+  font-weight: bold;
+  background: var(--default-elao-consent-primary);
+  border: none;
+  border-radius: 5px;
 }
 
 .elao-consent .message {
-    padding: 0 40px;
+  padding: 0 20px;
+  margin-bottom: 12px;
 }
 
 .elao-consent__details summary {
-    margin: 0 40px;
-    padding-right: 30px;
-    display: inline-block;
-    position: relative;
-    color: var(--default-elao-consent-primary);
-    font-weight: bold;
-    text-decoration: underline;
-    list-style: none;
+  margin: 0 20px;
+  padding-right: 30px;
+  display: inline-block;
+  position: relative;
+  color: var(--default-elao-consent-primary);
+  font-weight: bold;
+  text-decoration: underline;
+  list-style: none;
 }
 
 .elao-consent__details summary:before {
-    position: absolute;
-    top: 0;
-    right: 0;
-    font-size: 22px;
-    content: '🍪';
-    line-height: 1;
+  position: absolute;
+  top: 0;
+  right: -10px;
+  font-size: 30px;
+  content: '🍪';
+  line-height: 1;
 }
 
 .elao-consent__details ul {
-    padding: 20px 40px 0;
+  padding: 20px 40px 0;
```

Tu as passé le padding de `.message` à `0 20px`. Il faut réaligner le contenu du summary aussi.

```suggestion
  padding: 20px 20px 0;
```